src/test/java/com/automationPractice/steps/CustomerServiceSteps.java

```diff
@@ -22,4 +23,25 @@ public void doClickInTheContactUsButton() {
     public void theCUSTOMERSERVICEFromMustBeDisplayed() {
         Assert.assertTrue(contactUsPage.verifyThatPrincipalFormIsDisplayed(),"The principal form has not been displayed correctly");
     }
+
+    @Given("The user is in the CONTACT US section")
+    public void theUserIsInTheCONTACTUSSection() {
+        theUserBeInTheAutomationPracticePrincipalWebPage();
+        doClickInTheContactUsButton();
+    }
+
+    @When("The user click on subject heading control")
+    public void theUserClickOnSubjectHeadingControl() {
+        contactUsPage.doClickOnSubjectSelectControl();
+    }
+
+    @And("The user select the subject type {string}")
+    public void theUserSelectTheSubjectType(String Subject) {
+        contactUsPage.selectSubjectType(Subject);
+    }
+
+    @Then("Verify that description message {string} must be displayed")
+    public void verifyThatDescriptionMessageMustBeDisplayed(String Subject_desc_text) {
```

Comment:
Use the format used to name the params (Example: textToBeUsed)

Reply:
The parameter name has been changed to "textToBeShowed".